src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
@jhendrixMSFT  brought up a good point in that this will force us to support VS 2015 when building this project. I think this is a step that we are going to have to take eventually, but we might not be ready to do it know. Thoughts? @weshaggard .

I think for the "xunit.performance.core", which is the assembly that test projects actually reference, we may be able to get away with building against an older portable profile. The xunit.performance.execution assembly, which we don't reference directly here, probably needs to still be built as a v5.0 portable project. But that can probably be separated into a different Nuget package which we don't reference from individual test projects. Anyways, that would be quite a bit of fiddling just so that we don't have to use v5.0 portable projects.